pkg/types/ovirt/config.go

```diff
@@ -0,0 +1,76 @@
+package ovirt
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+var defaultOvirtConfigEnvVar = "OVIRT_CONFIG"
+var defaultOvirtConfigPath = filepath.Join(os.Getenv("HOME"), ".ovirt", "ovirt-config.yaml")
+
+// ErrCanNotLoadOvirtConfig
+var ErrCanNotLoadOvirtConfig error = errors.New("can not load ovirt config")
+
+// OvirtConfig holds oVirt api access details
```

Comment:
nit: comments should be complete sentences ending with `.` or proper punctuation.